qcschema/dev/properties/cc_properties.py

```diff
@@ -0,0 +1,101 @@
+"""
+The complete list of Coupled-Cluster properties.
+"""
+
+cc_properties = {}
+
+cc_properties["ccsd_same_spin_correlation_energy"] = {
+    "type":
+    "number",
+    "description":
+    "The portion of CCSD doubles correlation energy from same-spin (i.e. triplet) correlations, without any user scaling."
+}
+
+cc_properties["ccsd_opposite_spin_correlation_energy"] = {
+    "type":
+    "number",
+    "description":
+    "The portion of CCSD doubles correlation energy from opposite-spin (i.e. singlet) correlations, without any user scaling."
+}
+
+cc_properties["ccsd_singles_energy"] = {
```

Comment:
Inconsistency here where `ccsd_singles_energy` does not have `correlation` in it like `ccsd_opposite_spin_correlation_energy`. We have the same issue in MP2.

Reply:
Yes, I know. psi4 uses `_correlation`, but in a green-fields situation, I'd probably do a plain `ccsd_opposite_spin_energy` at this point and save `_correlation` for "total correlation". Either way justifiable, imo.